seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ConnectorPackageServiceIT.java

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
I find all test case running on same JVM instance. It can't prove the upload logic work fine. Please add some e2e test case with 1 client and 2 server, all runnning on different docker containers. You can refer `org.apache.seatunnel.e2e.common.container.seatunnel.SeaTunnelContainer`